packages/cinterop/src/jvm/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -495,6 +495,30 @@ actual object RealmInterop {
         )
     }
 
+    actual fun realm_object_changes_get_modified_properties(change: NativePointer): List<PropertyKey> {
+        val propertyCount = realmc.realm_object_changes_get_num_modified_properties(change.cptr())
+
+        val keys = LongArray(propertyCount.toInt())
+        realmc.realm_object_changes_get_modified_properties(change.cptr(), keys, propertyCount)
+        return keys.map { PropertyKey(it) }
+    }
+
+    actual fun realm_get_property(realm: NativePointer, className: String, propertyKey: PropertyKey): PropertyInfo {
```

Comment:
Yes it would be faster to fetch it from the schema, but with #669 we will get a major schema refactor, so I found easier to fetch it from JNI from the meantime.

I will create a ticket and tag this piece of code if 669 does not merge before this.